modules/serverless-logic-deploying-workflows-dev-mode.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-deploying-workflows-dev-mode"]
+= Deploying workflows in Dev mode
+
+You can deploy your local workflow on OpenShift in Dev mode. This allows you to experiment and modify your workflow directly on the cluster, seeing changes almost immediately. Dev mode is designed for development and testing purposes. It is ideal for initial development stages and testing new changes.
+
+.Prerequisites
+
+* {ServerlessLogicOperatorName} is installed on your cluster.
+* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
```

Comment:
again, there might be an attribute for "OpenShift Container Platform". And if there isn't one maybe add it?

Reply:
And I would change to just "You created"

Reply:
Added an attribute.